### ISSUE TITLE
[UX][D8] Views: Expose responsive classes in the UI via table settings

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#3656]
This is part of #378...

We already support setting `priority-medium` and `priority-low` in views settings, but we do that by manually adding the class (which a user needs to know exists) to the field:

<img width="564" alt="Screen Shot 2019-04-07 at 7 29 29 am" src="https://user-images.githubusercontent.com/2423362/55675506-08469f80-5907-11e9-82ba-2788b2b9b984.png">

D8 does it better:

![Screen Shot 2019-04-07 at 6 51 24 am](https://user-images.githubusercontent.com/2423362/55675509-14caf800-5907-11e9-9bf2-99a3738ddb1c.png)

**Comment by herbdool** [comment]
This is an old one, but we've finally got a solution. Not too difficult. (Borrowed a little bit from responsive_tables d7 module).

**Comment by herbdool** [comment]
Just found another reason for this approach rather than just a custom css class on a field label. Some fields, such as "Global: dropbutton" do not have an option for a custom label class, so there's no way to set the priority.

**Comment by herbdool** [comment]
The default views for admin_content and user_admin have been updated to use the new approach. So new sites, or reverting these views will make the page style options look like this:

<img width="1702" height="758" alt="Image" src="https://github.com/user-attachments/assets/de212f51-ae8c-45d0-8cbd-63b17a6fbd6e" />

**Comment by olafgrabienski** [comment]
The label "Responsive" isn't very clear in combination with values like _None_, _Low_, _Medium_ and _High_. @herbdool What do you think about "Responsive priority" instead?

**Comment by herbdool** [comment]
@olafgrabienski that works for me. Though we'd probably then want to change "None" to "High". Not that in the Backdrop version there was no "High" option.

**Comment by herbdool** [comment]
@olafgrabienski how about this

<img width="1698" height="758" alt="Image" src="https://github.com/user-attachments/assets/aff4cb29-6d36-4051-a4bd-0f4d97b66c25" />

The PR is updated.

**Comment by olafgrabienski** [comment]
Thank you very much – the new wording looks good to me!

(Unfortunately, I don't have time today to also test the functionality of the PR.)

**Comment by yorkshire-pudding** [comment]
@herbdool - I've given it a test and while it seems to work reliably with low priority, it is not working with Medium priority.  I set, on the Admin Content view, the following fields to Medium:

- Content Type
- Name
- Updated

This is the result when the width is reduced part way (Image stitched together from scrolling horizontally):
<img width="1037" height="577" alt="Image" src="https://github.com/user-attachments/assets/adc9d292-47ca-477b-b5e6-080b17dab9e1" />

The headings (Name and Updated) disappear but not the columns.

Once it gets narrow enough for Content Type to also disappear, then both headings and columns disappear and it works as expected.

**Comment by quicksketch** [comment]
@yorkshire-pudding I haven't been able to reproduce that problem. Did you have any fields set to low priority at the same time? I tested with a variety of medium and low priorities and they worked, including with sticky headers.

**Comment by herbdool** [comment]
I can't reproduce it either. Just tested on tugboat.

**Comment by docwilmot** [comment]
Same here, cannot reproduce, works for me.

**Comment by herbdool** [comment]
@docwilmot are you willing to update the labels?

**Comment by docwilmot** [comment]
Done, tested by me, assuming it's been code reviewed by @quicksketch.

**Comment by quicksketch** [comment]
Yep, implementation looks fine to me. Using the exact class names for the options seems a little sketchy but I can't see any problem with it. It could provide some flexibility for other options in the future besides just medium and low. Looks fine to me.

**Comment by quicksketch** [comment]
Merged https://github.com/backdrop/backdrop/pull/5241 into 1.x for 1.34.0. Thank you @herbdool, @docwilmot, @olafgrabienski, @yorkshire-pudding, and @klonos!

**Comment by olafgrabienski** [comment]
>> while it seems to work reliably with low priority, it is not working with Medium priority

> I haven't been able to reproduce that problem.

Today I was able to reproduce the issue with Medium priority at the `admin/content` view and filed a report: https://github.com/backdrop/backdrop-issues/issues/7126
